README.md

```diff
@@ -79,6 +80,12 @@ In case you want to create the index on a source table, refer to the table using
 ) }}
 ```
 
+#### dateadd ([source](macros/multiple_databases/dateadd.sql))
+This macro adds a specified number value (as a signed integer) to a specified datepart of an input date value, and then returns that modified value. The datepart can be any of the following values for SQL Server and Snowflake: year, quarter, month, week, day, hour, minute, second, millisecond. For Databricks, the datepart can be any of the following values: year, day, hour, minute, second.
```

Comment:
We are expecting a date or a datetime as input for this function, right? Can this be mentioned here explicitly. Now it states "input date value" and in the example uses "expression" (which can be a string)